package.json

```diff
@@ -131,8 +131,8 @@
     "xgettext-template": "^3.4.2"
   },
   "engines": {
-    "node": "8.* || >= 10.*",
-    "yarn": ">= 1.0.0"
+    "node": ">= 10.*",
```

Comment:
Why not 12

Reply:
@iamareebjamal I am unable to make netlify utilise the latest version as it is using the cached version of node which is 10 used in recent PR builds

Reply:
@iamareebjamal Ping!